.github/workflows/tests.yml

```
@@ -104,6 +108,9 @@ jobs:
             mamba
             fmt!=10.2.0
         if: ${{ !startswith(matrix.kind, 'pip') }}
+      # Make sure we have the right Python
+      - run: python -c "import platform; platform.machine() == 'arm64', platform.machine()"
```

is there a missing `assert` on this line?

```suggestion
      - run: python -c "import platform; assert platform.machine() == 'arm64', platform.machine()"
```